pilot/lib/pilot/build_manager.rb

```diff
@@ -61,7 +61,8 @@ def distribute(options, build = nil)
           return
         end
 
-        UI.message("Distributing build #{build.train_version}(#{build.build_version}) from #{build.testing_status} -> External")
+        type = options[:distribute_external] ? 'External' : 'Internal'
+        UI.message("Distributing build #{build.train_version}(#{build.build_version}) from #{build.testing_status} -> #{type}")
```

Comment:
I also took note that spaceship doesn't seem to handle internal and external builds in the same way:
```ruby
      # @param (testing_type) internal or external
      def update_testing_status!(new_value, testing_type, build = nil)
        build ||= latest_build if testing_type == 'external'
```

But this isn't a problem here as the build is specified. Still weird though.